lib/spack/spack/build_systems/oneapi.py

```diff
@@ -22,46 +24,52 @@ class IntelOneApiPackage(Package):
 
     phases = ['install']
 
-    def component_info(self,
-                       dir_name,
-                       components,
-                       releases,
-                       url_name):
+    # oneAPI license does not allow mirroring outside of the
+    # organization (e.g. University/Company).
+    redistribute_source = False
+    
```

Comment:
According to style checks, this line contains whitespace.

Reply:
Fixed all ```spack style``` issues.